index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="main">
```

Comment:
I think it would be better to use class in div instead of id.

Reply:
- [ ]  Always make sure to include an explanation, a "why", for each requested change.